pkg/auth/dependency/newinteraction/nodes/remove_identity.go

```diff
@@ -0,0 +1,72 @@
+package nodes
+
+import (
+	"github.com/authgear/authgear-server/pkg/auth/dependency/identity"
+	"github.com/authgear/authgear-server/pkg/auth/dependency/newinteraction"
+	"github.com/authgear/authgear-server/pkg/core/authn"
+)
+
+func init() {
+	newinteraction.RegisterNode(&NodeRemoveIdentity{})
+}
+
+type InputRemoveIdentity interface {
+	GetIdentityType() authn.IdentityType
+	GetIdentityID() string
+}
+
+type EdgeRemoveIdentity struct{}
```

Comment:
As discussed, we need to add `IdentityType` and `IdentityID` in the edge. Let me fix this in removing anonymous identity.